packages/components/src/tabs/src/Tabs.css

```diff
@@ -12,70 +12,105 @@
+    font-size: var(--hop-body-sm-font-size);
     position: relative;
-    padding: var(--o-ui-text-button-padding-md);
-    height: var(--o-ui-sp-7);
+    height: 2rem;
     flex-shrink: 0;
 }
 
+.o-ui-tab-list-popover-trigger:hover {
+    color: var(--hop-neutral-text);
+}
+
+/* TAB LIST | POPOVER | TRIGGER | ORIENTATION | HORIZONTAL */
+.o-ui-tabs-horizontal .o-ui-tab-list-popover-trigger {
+    border-left: 1px solid var(--hop-neutral-border-weak);
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    margin-right: var(--hop-space-inline-sm);
+}
+
+/* TAB LIST | POPOVER | TRIGGER | ORIENTATION | VERTICAL */
+.o-ui-tabs-vertical .o-ui-tab-list-popover-trigger {
+    border-top: 1px solid var(--hop-neutral-border-weak);
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    margin-left: var(--hop-space-inline-xs);
+    margin-bottom: var(--hop-space-stack-sm);
+}
+
 /* TAB LIST | POPOVER | TRIGGER | OPEN */
 .o-ui-tab-list-popover-trigger[aria-expanded="true"] {
-    background-color: var(--o-ui-bg-alias-basic-transparent-hover);
+    background-color: var(--hop-neutral-surface-weakest-hover);
 }
 
 /* TAB LIST | POPOVER | TRIGGER | HOVER */
 .o-ui-tab-list-popover-trigger:not([disabled]):hover {
-    background-color: var(--o-ui-bg-alias-basic-transparent-hover);
+    background-color: var(--hop-neutral-surface-weakest-hover);
 }
 
 /* TAB LIST | POPOVER */
 .o-ui-tab-list-popover {
-    background-color: var(--o-ui-bg-alias-surface);
-    border: 1px solid var(--o-ui-b-alias-low-break);
-    border-radius: var(--o-ui-br-3);
-    box-shadow: var(--o-ui-bs-alias-floating);
-    margin-top: var(--o-ui-sp-2);
+    width: max-content;
+    max-width: 20rem;
+    overflow-y: auto;
+    background-color: var(--hop-neutral-surface);
+    outline: transparent;
+    border: 1px solid var(--hop-neutral-border-weak);
+    border-radius: var(--hop-shape-rounded-md);
+    box-shadow: var(--hop-elevation-lifted);
+    padding: var(--hop-space-inset-sm);
 }
 
 /* TAB LIST | POPOVER | TAB */
 .o-ui-tab-list-popover .o-ui-tab {
-    padding-left: calc(var(--o-ui-sp-3) + 1px);
-    padding-right: var(--o-ui-sp-8);
+    color: var(--hop-neutral-text);
+    border-radius: var(--hop-shape-rounded-md);
+    font-size: var(--hop-body-sm-font-size);
+    height: 2rem;
+    min-height: 2rem;
+    padding: 0 var(--hop-space-inset-md);
+    text-align: left;
+    display: flex;
+    align-items: center;
+    justify-items: center;
     justify-content: start;
+    position: relative;
     width: 100%;
-    min-height: var(--o-ui-sz-7);
 }
 
 /* TAB LIST | POPOVER | TAB */
 .o-ui-tab-list-popover .o-ui-tab:not([disabled]):hover {
-    background-color: var(--o-ui-bg-alias-mid-break);
+    background-color: var(--hop-neutral-surface-hover);
     border-bottom: 0;
 }
 
 /* TAB LIST | POPOVER | FOCUS */
 .o-ui-tab-list-popover .o-ui-tab::before {
-    left: 8px;
-    right: 8px;
+    left: 0.5rem;
+    right: 0.5rem;
```

Comment:
I would use var(--hop-space-inline-sm) for these.

Reply:
It's fixed in the Tab PR